src/Integration.Vsix/Helpers/IIssueConverter.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace SonarLint.VisualStudio.Integration.Vsix.Helpers
 {
-    public interface IIssueConverter
+    internal interface IIssueConverter
```

Comment:
Changed to _internal_ for consistency (none of the tagger/tracker classes need to be public and most of them are _internal_).